cylc/uiserver/workflows_mgr.py

```diff
@@ -62,7 +65,7 @@
         except socket.error as exc:
             if flags.debug:
                 raise
-            sys.stderr.write("ERROR: %s: %s\n" % (exc, host))
+            logger.error("ERROR: %s: %s\n" % (exc, host))
```

Comment:
```suggestion
            logger.error("%s: %s", exc, host)
```

(Remove `ERROR: ` and don't format the string upfront for lazy logging.)